spec/index.emu

```diff
@@ -38,19 +38,35 @@ location: https://rawgit.com/tc39/proposal-realms/master/index.html
   <h1>Realm Objects</h1>
   <emu-clause id="sec-realm-abstract-operations">
     <h1>Realm Abstract Operations</h1>
-    <emu-clause id="sec-performRealmEvaluation" aoid="PerformRealmEvaluation">
-      <h1>PerformRealmEvaluation ( _x_, _evalRealm_ )</h1>
+
+    <emu-clause id="sec-perform-realm-script-evaluation" aoid="PerformRealmScriptEvaluation">
+      <h1>PerformRealmScriptEvaluation ( _sourceText_, _realm_ )</h1>
       <emu-alg>
-      1. Assert: Type(_x_) is String.
+      1. Assert: _sourceText_ is an ECMAScript source text (see clause <emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>).
       1. Assert: _evalRealm_ is a Realm Record.
       1. Let _hostDefined_ be _evalRealm_.[[HostDefined]].
-      1. Let _s_ be ParseScript(_x_, _evalRealm_, _hostDefined_).
+      1. Let _s_ be ParseScript(_sourceText_, _realm_, _hostDefined_).
       1. If _s_ is a List of errors, then
         1. Perform HostReportErrors(_s_).
         1. Return NormalCompletion(*undefined*).
       1. Return ? ScriptEvaluation(_s_).
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-perform-realm-module-evaluation" aoid="PerformRealmModuleEvaluation">
+      <h1>PerformRealmModuleEvaluation ( _sourceText_, _realm_ )</h1>
+      <emu-alg>
+      1. Assert: _sourceText_ is an ECMAScript source text (see clause <emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>).
+      1. Assert: _realm_ is a Realm Record.
+      1. Let _hostDefined_ be _evalRealm_.[[HostDefined]].
+      1. Let _m_ be ParseModule(_sourceText_, _realm_, _hostDefined_).
+      1. If _m_ is a List of errors, then
+        1. Perform HostReportErrors(_m_).
+        1. Return NormalCompletion(*undefined*).
+      1. Assert: All dependencies of _m_ have been transitively resolved and _m_ is ready for evaluation.
+      1. Return ? _m_.Evaluate().
```

Comment:
You probably don't want to replace _m_.Evaluate(), as this is undefined for all Cyclic Module records (which this will be). ([spec](https://tc39.es/ecma262/#sec-moduleevaluation)).

Instead, maybe you want to perform ? _m_.Evaluate() and then return GetModuleNamespace(_m_).